### ISSUE TITLE
Develop integration script in shiny for recruitment, yellow and silver data

### ISSUE THREAD

**Issue by cedricbriandgithub** [ices-eg/wg_WGEEL#78]
currently we use an external interface, time to unify everything

**Comment by cedricbriandgithub** [comment]
task list

* UI.R simply add type
![image](https://user-images.githubusercontent.com/26055877/90503268-6a482380-e14f-11ea-8f9a-5dd66eb5899a.png)

 * database_data.R  function extract data to be adapted add three type
```
	list_data_table = data.frame(data_needed = 
            c("Landings", "Aquaculture", "Release", "B0", "Bbest", "Bcurrent", "Sigma A",
                 "Sigma F", "Sigma H", "Potential available habitat", "Mortality in Silver Equivalents", 
                 "Sigma F all", "Sigma H all", "PrecoData Country", "PrecoData EMU","PrecoData All"), 
         table_dbname = c("landings", "aquaculture", "release", "b0", "bbest", "bcurrent", "sigmaa", 
             "sigmaf", "sigmah", "potential_available_habitat","silver_eel_equivalents", "sigmafallcat", 
             "sigmahallcat", "precodata_country", "precodata_emu","precodata_all"))
```

Biometry data
UI 
 but check types I this depends on database structure. See discussion here https://github.com/ices-eg/wg_WGEEL/issues/104

**Comment by elfunesto** [comment]
In the closed issue #91, Laurent made a reference "see standard_2019.R as a starting point" that should still be valid

**Comment by cedricbriandgithub** [comment]
test done, test on server now
